README.md

```diff
@@ -111,17 +111,21 @@ This will start listening on the provided port, blocking until a client makes a
 
 ### Tracing
 
-As you are implementing, you may want to trace incoming and outgoing messages. Initialize the server with `:trace? true` and then read traces (two element vectors, beginning with the log level `:debug` and ending with a string, the trace itself) off its `:trace-ch`.
+As you are implementing, you may want to trace incoming and outgoing messages. Initialize the server with `:trace-level "verbose"` and then read traces (two element vectors, beginning with the log level `:debug` and ending with a string, the trace itself) off its `:trace-ch`.
 
 ```clojure
-(let [server (lsp4clj.io-server/stdio-server {:trace? true})]
+(let [server (lsp4clj.io-server/stdio-server {:trace-level "verbose"})]
   (async/go-loop []
     (when-let [[level trace] (async/<! (:trace-ch server))]
       (logger/log level trace)
       (recur)))
   (lsp4clj.server/start server context))
 ```
 
+`:trace-level` can be set to `"off"` (no tracing), `"messages"` (to show just the message time, method, id and direction), or `"verbose"` (to also show details of the message body).
```

Comment:
nit: maybe the value could be a keyword? `:messages`, `:off` etc

Reply:
My thinking was that keywords would be more Clojure-y, but since the spec defines these 3 values as strings, that’s what language servers will get from clients. It seems like an extra step to make them `(keyword trace)`. I’d be ok with either—what would you prefer?

Reply:
Makes sense, I think string would be better indeed